### PR TITLE
Response

### PR DESCRIPTION
**Pull request by faetea**
I used my Ref.txt (my notes of useful commands)
also hadn't realized I needed to select the Branch: Response before trying to do the pull req. So I had to ask Kelvin

### CODE REVIEW

File: challenge.md [hunk 1]
```diff
@@ -5,19 +5,22 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+A: $ git branch tps --set-upstream-to origin
```
Comment (GA-MEB) [inline challenge.md:8]: No, sorry. The command we'd want to use to copy our fork to our local machine is `git clone` (specifically, `git clone {ssh-url}`)

File: challenge.md [hunk 2]
```diff
@@ -5,19 +5,22 @@
...
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+A: $ git status
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
+A: $ git log
 
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
+A: $ git -b <branchName>
```
Comment (GA-MEB) [inline challenge.md:17]: Not quite - it's `git checkout -b <branchName>`

File: challenge.md [hunk 3]
```diff
@@ -5,19 +5,22 @@
...
 
-
-5. I want to add a file to the staging area. What git commit would I use?
-
+5. I want to add a file to the staging area. What git command would I use?
+A: $ git add <file.txt>
```
Comment (GA-MEB) [inline challenge.md:20]: Strictly, it's not just the name of the file you want to add - it's a _path_ to the file.

File: challenge.md [hunk 4]
```diff
@@ -5,19 +5,22 @@
...
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
+A: $ git pull // will download things from other repository and then merge the development histories
```
Comment (GA-MEB) [inline challenge.md:23]: We're actually talking about a single repo here, not sharing changes between repos.

We've made a whole bunch of changes on the `add-directories` branch, and now we want to incorporate those changes into the `master` branch. To do this we first need to move to the `master` branch (`git checkout master`) and then run `git merge {branch-to-get-changes-from}` (in this case, `add-directories`).

**Comment by GA-MEB** [comment]
3.5 of 6

Let me know if you have any questions - happy to walk through this with you!

**Comment by faetea** [comment]
@MattB-GA-Boston Thank you Matt.  I think it mostly makes sense to me now.  It just took some practice.

**Pull request closed without merging**